packages/buttons/src/elements/Anchor.tsx

```diff
@@ -10,12 +10,11 @@ import PropTypes from 'prop-types';
 import { StyledAnchor, StyledExternalIcon } from '../styled';
 
 export interface IAnchorProps extends AnchorHTMLAttributes<HTMLAnchorElement> {
-  /** Apply danger styling */
+  /** Determines if danger styling is used */
```

Comment:
@jzempel Is the passive voice preferred here? It looks like there is a nuance here that I might not understand, so I will defer to you on this, but UXCS tends to use active voice: This does that. `Applies danger styling`

Reply:
@vsrixyz We can change all the boolean values to "Applies", I chose "Determines" to help communicate the if/else quality of the property. Either way, let me know what's preferable

Reply:
I'd defer to @jzempel on this one, he's closer to the target audience so I'd like to get his take.

Reply:
We want to follow the UXCS and `website` content guidance to use active voice. @KICKdesigns please scan through the current PR set and double-check passive->active voice updates. :pray:

#924, #926, #928 (current), #930, #931, #932

Reply:
Fixed in latest push.